src/content/en/agencies/js/directory.js

```diff
@@ -0,0 +1,656 @@
+/**
+ * @fileoverview Creates web directory of agencies based on JSON data
+ * TODO: Eventually refactor this to be used as a 'library' for a variety of directories
+ */
+
+/**
+ * Initializing the entire directory
+ *
+ * @param {String} markup, defined in script in HTML file
+ * Example: "<div><% this.example_data %></div>"
+ *
+ * @param {String} requestUrl, path to relevant directory JSON file
+ */
+
+var initializeDirectory = function(markup, requestUrl, alphabeticalSortKey) {
+
+  /**
+  *
+  * Directory-level vars
+  *
+  */
+
+  // Element vars
+  var frameboxEl = document.querySelector('.directory-framebox');
+  var frameboxWrapperEl = document.querySelector('.directory-framebox__wrapper');
+  var cardsWrapperEl;
+  var filterEls;
+
+  // Data vars
+  var cardsData;
+  var filtersData;
+  var modifiedCardsData;
+  var checkedFilters = {};
+  var searchQuery = '';
+
+  // Map vars
+  var infoWin;
+  var map;
+  var markers = [];
+
+ /**
+ *
+ *
+ * Helper functions
+ *
+ *
+ */
+
+  /**
+   * A basic templating engine, uses <% %> as its preferred delimiters
+   *
+   * @param {String} html
+   * Example: "<div><% this.example_data %></div>"
+   *
+   * @param {Object} data
+   * Example: { "example_data" : "print this" }
+   *
+   */
+  var TemplateEngine = function(html, data) {
+    var re = /<%([^%>]+)?%>/g, reExp = /(^( )?(if|for|else|switch|case|break|{|}))(.*)?/g, code = 'var r=[];\n', cursor = 0, match;
```

Comment:
> 80 chars

Reply:
Going to nix the template engine based on your comment below